### PR TITLE
Making changes to finish signup proccess

### PR DESCRIPTION
**Pull request by andresgutgon**
## WAT
I'm making the screen on client app to complete signup when user receives email confirmation. This PR is to make all necessary changes

### CODE REVIEW

File: engines/account/app/mailers/account/signup_mailer.rb [hunk 1]
```diff
@@ -17,7 +17,7 @@ def confirm_email(signup)
     #
     # @return [String]
     def confirm_url
-      "http://10.0.3.70:8000?token=#{@signup.token}"
+      "http://10.0.3.70:8000/signup/complete/#{@signup.token}"
```
Comment (andresgutgon) [inline engines/account/app/mailers/account/signup_mailer.rb]: Better URL

**Comment by enricostano** [comment]
We'll need to merge this changes with https://github.com/coopdevs/katuma/pull/55

**Comment by andresgutgon** [comment]
@enricostano close this PR and cherry-pick this change

**Comment by enricostano** [comment]
I will, but we need a better way to manage non API urls

**Comment by andresgutgon** [comment]
> but we need a better way to manage non API urls in Rails

A small lib with all frontend urls

**Comment by enricostano** [comment]
yep, but I still need to find out how to avoid hardcoded host information. E.g. `localhost` vs `10.0.3.70` vs `katuma.org`

**Comment by andresgutgon** [comment]
> yep, but I still need to find out how to avoid hardcoded host information

Enviroment variable checking when initializing that lib, check `lib/front_end_url.rb` in well,...you know :trollface:

**Comment by enricostano** [comment]
eheh.. I was looking for a more Railsy way... whatever that means.... XD

**Comment by andresgutgon** [comment]
Cherry picked

**Pull request closed without merging**